src/renderer/src/store/reducers/AuthReducer.ts

```diff
@@ -110,13 +107,34 @@ const authReducerMigrations = {
       },
     }
   },
+  3: (state: any) => {
+    const applicationDataByLoginType = Object.keys(state.data.applicationDataByLoginType).reduce((acc, key) => {
+      const loginType = key as TLoginSessionType
+      const applicationData = state.data.applicationDataByLoginType[loginType]
+
+      acc[loginType] = {
+        ...applicationData,
+        notifications: [],
+      }
+
+      return acc
+    }, {} as TApplicationDataByLoginType)
+
+    return {
+      ...state,
+      data: {
+        ...state.data,
+        applicationDataByLoginType,
+      },
+    }
+  },
```

Comment:
Is it really necessary? You are adding the notifications array in the config.

Reply:
Sorry, I didn't understand

Reply:
It is necessary to insert the notifications array inside the auth storage, otherwise, when we try to save a notification in an already filled storage, it will not exist.